src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -214,6 +215,12 @@ public bool IgnoreTargetFrameworkAttributeVersionMismatch
         /// </remarks>
         public bool FindDependenciesOfExternallyResolvedReferences { get; set; }
 
+        /// <summary>
+        /// If true, outputs any unresolved assembly conflicts (MSB3277) in UnresolvedAssemblyConflicts
+        /// instead of logging them.
```

Comment:
> instead of logging them.

doesn't it do both?

Reply:
Yep! Fixed. That was a holdover from how I'd originally designed it.